src/toil/batchSystems/torque.py

```diff
@@ -121,6 +121,15 @@ def prepareQsub(self, cpu, mem, jobID):
             if cpu is not None and math.ceil(cpu) > 1:
                 qsubline.extend(['-l ncpus=' + str(int(math.ceil(cpu)))])
 
+            # "Native extensions" for TORQUE (see DRMAA or SAGA)
+            nativeConfig = os.getenv('TOIL_TORQUE_ARGS')
+            if nativeConfig is not None:
+                logger.debug("Native TORQUE options appended to qsub from TOIL_TORQUE_RESOURCES env. variable: {}".format(nativeConfig))
+                #if ("--mem" in nativeConfig) or ("--cpus-per-task" in nativeConfig):
+                if ("mem=" in nativeConfig) or ("nodes=" in nativeConfig) or ("ppn=" in nativeConfig):
+                    raise ValueError("Some resource arguments are incompatible: {}".format(nativeConfig))
```

Comment:
Error message would be even more helpful if it listed the arguments that are not aloud.